File: Toro.Domain/Entity/Patrimony.cs
using System.Collections.Generic;

namespace Toro.Domain.Entity {
    public class Patrimony : To<int> {

        public int InvestorId { get; private set; }
        public Investor Investor { get; private set; }
        public decimal AccountAmount { get { return AccountAmount; } set { this.AccountAmount += value; } }
        public IList<AssetXPatrimony> AssetXPatrimony { get; set; }

        private Patrimony() {
        }
    }
}

File: app/src/app/model/patrimony.ts
export class Patrimony {
    id: number;  
    investorId: number;  
    accountAmount: number;  
    
    constructor(id: number, investorId: number, accountAmount: number){
        this.id = id;
        this.investorId = investorId;
        this.accountAmount = accountAmount;
    }
}